src/pip/_internal/commands/configuration.py

```diff
@@ -206,19 +205,21 @@ def list_config_values(self, options, args):
             for fname in files:
                 with indent_log():
                     file_exists = os.path.exists(fname)
-                    write_output("%s, exists: %r",
-                                 fname, file_exists)
+                    write_output("%s, exists: %s",
+                                 fname, str(file_exists))
```

Comment:
nit: `str()` is redundant

Reply:
I added `str` since `write_output` takes input as `args` as `str`

https://github.com/pypa/pip/blob/4bfc54df9bbceda24393405d3360411da90a9f00/src/pip/_internal/utils/misc.py#L570-L571

and `file_exists` just above this returns a bool

https://github.com/pypa/pip/blob/4bfc54df9bbceda24393405d3360411da90a9f00/src/pip/_internal/commands/configuration.py#L208

So the cast avoid mypy complaining about casting `bool` to `str`.

Is there a better approach to avoid this ?

Reply:
Ah I see. Maybe it would be more intuitive if `write_output` had the same type signature as `logger.info`?

Reply:
I agree. Do you suggest me making that change in this PR, or leaving this cast as `str` and making the signature change in a follow-up?

Reply:
As you prefer.

Reply:
On further thought, I have added https://github.com/pypa/pip/pull/8365/commits/fb057308981225f5fbca9fcbc9e3d6c8eb88c99b to align the type to [logger.info](https://github.com/python/typeshed/blob/master/stdlib/2and3/logging/__init__.pyi#L77)